Vagrantfile

```diff
@@ -48,8 +48,7 @@ Vagrant.configure("2") do |config|
          "all_groups:children" => ["rs"]
        }
        ansible.extra_vars = {
-
-         selinux: false,
+         selinux: true,
```

Comment:
So retrace-server doesn’t work properly without SELinux in permissive mode, but doesn’t this set it to enforcing?

Reply:
Oh, my bad, I missed the `selinux_state` variable. Carry on.